Python/RSSFeedReader/RSSFeedReader.py

```diff
@@ -1,9 +1,18 @@
 # RSSFeedReader
 
-# Set URL for Feed
+# Requests library used for get()
+import requests
 
-# Request RSS Feed for set URL and collect output
-
-# print response output
+# Set the RSS FeedURL
+URL = 'http://www.motortrend.com/widgetrss/motortrend-stories.xml'
 
+# Request RSS Feed for set URL and collect output
+try:
+  response = requests.get(URL)
+  response.raise_for_status()
```

Comment:
What does this line do?

Reply:
I am talking about line 12

Reply:
I found documentation here: https://2.python-requests.org/en/master/user/quickstart/
- No MSDN type page yet ;-)

It is used to raise the http_error.
I tried to cause and use the "raise" output and nothing seemed to work.
I did however add error handling using .status_code

Reply:
Here is the msdn page...

https://docs.python.org/3/library/index.html

Reply:
Looks like "requests" is a third party library.  It looks like there are core objects that you can use for an http request. I would try using those instead to start out with.

Reply:
check this page...

https://docs.python.org/3/library/urllib.request.html#module-urllib.request

Reply:
That would explain why I could not find requests on this page ;-)
I will re-code with this native urllib.request